### PR TITLE
add forest area

### PR DESCRIPTION
**Pull request by IlijazM**
Closes Gamify-IT/issues#41.

### CODE REVIEW

**Comment by delvh** [comment]
image
Ouch.

File: ProjectSettings/Physics2DSettings.asset [hunk 1]
```diff
@@ -53,4 +53,4 @@ Physics2DSettings:
   m_ColliderAsleepColor: {r: 0.5686275, g: 0.95686275, b: 0.54509807, a: 0.36078432}
   m_ColliderContactColor: {r: 1, g: 0, b: 1, a: 0.6862745}
   m_ColliderAABBColor: {r: 1, g: 1, b: 0, a: 0.2509804}
-  m_LayerCollisionMatrix: ffffffffffffffffffffffffffffffffffffffffffffffff7fffffffbfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
+  m_LayerCollisionMatrix: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
```
Comment (delvh) [inline ProjectSettings/Physics2DSettings.asset:56]: Ah yes. Such a difference.

File: Assets/Scenes/Forrest/MagicTree.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(SpriteRenderer))]
+public class MagicTree : MonoBehaviour
```
Comment (delvh) [inline Assets/Scenes/Forrest/MagicTree.cs:6]: Perhaps you should a small documentation for this class?

**Review by delvh — CHANGES_REQUESTED** [review]
You misspelled `forest`.

**Review by leonleoff — APPROVED** [review]

**Review by WuestFlorian — APPROVED** [review]

**Review by Aaron2000119 — APPROVED** [review]

File: Assets/Scenes/Forrest/MagicTree.cs (hunk 2 above)
Comment (leonleoff) [reply Assets/Scenes/Forrest/MagicTree.cs]: @delvh  I corrected the error.

**Pull request merged**